tests/test_salt_matching.py

```diff
@@ -6,16 +6,45 @@
 salt_ion_match.py
 """
 
+import logging
 import platform
+from itertools import product
 
 import numpy as np
 import pytest
 
 import pyEQL
 from pyEQL.salt_ion_match import Salt
 
+_ANIONS = ["Cl[-1]", "SO4[-2]", "PO4[-3]", "ClO[-1]", "NO3[-1]", "CO3[-2]", "MnO4[-2]"]
+_CATIONS = ["Na[+1]", "Ca[+2]", "Fe[+3]", "K[+1]", "Li[+1]", "Cu[+2]", "Ba[+2]"]
+_SALTS = list(product(_CATIONS[:3], _ANIONS[:3]))
+_CONJUGATE_BASES = [
+    ("HCO3[-1]", "CO3[-2]"),
+    ("H2PO4[-1]", "PO4[-3]"),
+]
+_CONJUGATE_BASE_PAIRS = [
+    ((cation, conjugates[0]), (cation, conjugates[1])) for cation, conjugates in product(_CATIONS[:3], _CONJUGATE_BASES)
+]
 
-def test_salt_init():
+
+@pytest.fixture(name="cutoff", params=[1e-8])
+def fixture_cutoff(request: pytest.FixtureRequest) -> float:
+    return float(request.param)
+
+
+@pytest.fixture(name="use_totals", params=[True])
+def fixture_use_totals(request: pytest.FixtureRequest) -> bool:
+    return bool(request.param)
+
+
+@pytest.fixture(name="salt_dict")
+def fixture_salt_dict(solution: pyEQL.Solution, cutoff: float, use_totals: bool) -> dict[str, dict[str, float | Salt]]:
+    salt_dict: dict[str, dict[str, float | Salt]] = solution.get_salt_dict(cutoff=cutoff, use_totals=use_totals)
+    return salt_dict
+
+
+def test_salt_init() -> None:
```

Comment:
Let's add one more test to this function that evaluates a 2:2 salt like MgCl2. It should result in `nu_cation` and `nu_anion` both being 1